distributed/client.py

```diff
@@ -2779,6 +2779,17 @@ def scatter(
                 "Consider using a normal for loop and Client.submit"
             )
 
+        if broadcast and dask.config.get(
+            "distributed.scheduler.active-memory-manager.start"
+        ):
```

Comment:
This is problematic.
1. you want to specifically test for ReduceReplicas. Namely, at the moment of writing AMM also serves the RetireWorker policy and could serve other, harmless user-defined policies.
2. the AMM could have been started by hand with `client.amm.start()`. The API call `clent.amm.running()` is there for this.

I would instead suggest adding a new RPC endpoint to 
- `distributed.active_memory_manager.AMMClientProxy` and to
- `distributed.active_memory_manager.ActiveMemoryManagerExtension.amm_handler`

which returns a set of currently running policies, and test that instead.
(caveat: there can be more than one RetireWorker policies running when you have more than one worker closing gracefully at the same time).

Reply:
Thanks @crusaderky - I agree that the config check is not sufficient. Thanks for the suggestions!